### ISSUE TITLE
splash screen invisible

### ISSUE THREAD

**Issue by lemzwerg** [fontforge/fontforge#3787]
[da431519bab799cb76a1c199b9e615e81ad31c3a]
[openSUSE Leap 15.1]

Configuring fontforge with
```
./configure --enable-debug-raw-points \
            --enable-silent-rules \
            --enable-freetype-debugger=... \
            CFLAGS="-g -O0"
```
(this results in not using GTK), followed by calling `make && make install`, then starting FontForge without argument produces an empty splash screen.  Is this expected?  I have all necessary graphics libraries installed (only missing libspiro, whch shouldn't be relevant here).

**Comment by jtanx** [comment]
Ugh this hurts my brain, so it seem like this is a bit of a bug only in the gxdraw/non-gdk backend.

If only an image is displayed, and no text rendered, it doesn't actually get flushed to the screen. When text does get written, the image finally gets shown.

The text doesn't get rendered on startup because of a change in #3785, due to another glitch in the text all being rendered at the bottom of the screen.

Thinking about it a bit more, that glitch seen in #3785 is probably because as/fh were 0 when the splash started (since those metrics are computed after), and because we removed the resizing during startup, it only ever gets rendered once.

It's getting late, so I'll look into this some other time, unless someone else fixes this first.

**Comment by ctrlcctrlv** [comment]
Hi @lemzwerg - can you please try commit https://github.com/fontforge/fontforge/commit/a508d470f0093e9768199b3414a7cb41d457670b to see if the recent commit by @jtanx (https://github.com/fontforge/fontforge/commit/93605760bda7a318d925a35bbf194a4ea6468ed2) caused this issue?

**Comment by jtanx** [comment]
Ok that's not quite correct, the changes in #3785 to not render the text don't cause the issue. It seems there's a requirement for the resize event to make it display properly (which used to happen when it did that resizing thing on the timer).

I also did actually manage to cobble a hack together.

**Comment by jtanx** [comment]
Fixed by #3788
